ui/app/components/app/sidebars/sidebar.component.js

```diff
@@ -30,12 +30,12 @@ export default class Sidebar extends Component {
 
   renderSidebarContent() {
     const { type, sidebarProps = {} } = this.props;
-    const { transaction = {} } = sidebarProps;
+    const { transaction = {}, onSubmit } = sidebarProps;
```

Comment:
I'm having trouble following where `onSubmit` is set :thinking: 

It looks like the `sidebarProps` are only set in one place: `useRetryTransaction.js` on line 46, which does not specify an `onSubmit` prop.

Reply:
Nevermind, I see now. One more call was introduced in `useCancelTransaction`. Nevermind!